src/dialogs/vkbasalt.py

```diff
@@ -0,0 +1,306 @@
+        if True in [
+            self.cas.get_enable_expansion(),
+            self.dls.get_enable_expansion(),
+            self.fxaa.get_enable_expansion(),
+            self.smaa.get_enable_expansion(),
+            (self.lut_file_path and self.clut.get_enable_expansion())
+        ]:
+            self.btn_save.set_sensitive(True)
+        else:
+            self.btn_save.set_sensitive(False)
+
+    def __default(self, widget, state):
+        self.cas.set_sensitive(not state)
+        self.dls.set_sensitive(not state)
+        self.fxaa.set_sensitive(not state)
+        self.smaa.set_sensitive(not state)
+        self.row_disable_on_launch.set_sensitive(not state)
+        self.clut.set_sensitive(not state)
+        if state is False:
+            if self.cas.get_enable_expansion() is False and self.dls.get_enable_expansion() is False and self.fxaa.get_enable_expansion() is False and self.smaa.get_enable_expansion() is False and self.clut.get_enable_expansion() is False:
+                self.btn_save.set_sensitive(False)
+        else:
+            self.btn_save.set_sensitive(True)
+
+    def __change_edge_detection_type(self, widget, edge_detection_type):
+        self.smaa_edge_detection = edge_detection_type
+        self.luma.handler_block_by_func(self.__change_edge_detection_type)
+        self.color.handler_block_by_func(self.__change_edge_detection_type)
+        if edge_detection_type == "luma":
+            self.color.set_active(False)
+            self.luma.set_active(True)
+        elif edge_detection_type == "color":
+            self.color.set_active(True)
+            self.luma.set_active(False)
+
+        self.luma.handler_unblock_by_func(self.__change_edge_detection_type)
+        self.color.handler_unblock_by_func(self.__change_edge_detection_type)
+
+    def __import_clut(self, *args):
+        def set_path(_dialog, response, _file_dialog):
+            if response == -3:
+                self.lut_file_path = _file_dialog.get_file().get_path()
+
+                if self.lut_file_path.split(".")[-1] == "png":
+
+                    texture = Gdk.Texture.new_from_filename(self.lut_file_path)
+
+                    width = texture.get_width()
+                    height = texture.get_height()
+
+                    def error_dialog(title, message):
+                        dialog = Adw.MessageDialog.new(self.window, title, message)
+                        dialog.add_response("cancel", "Close")
+                        dialog.present()
+
+                    def set_lut_file_path():
+                        if self.input_entry.get_text():
+                            self.lut_file_path = self.input_entry.get_text()
+                        else:
+                            self.lut_file_path = False
+
+                    if " " in self.lut_file_path:
+                        error_dialog("Spaces in File Name", "Color Lookup Table path must not contain any spaces. Please rename the file to remove all spaces.")
```

Comment:
These should be translatable strings. Use `_("Some translatable string")`. Since you added new translatable strings you have to add the file name to `po/POTFILES` (also the UI file).